tools/openvino_dev/setup.py

```diff
@@ -252,7 +252,7 @@ def read_requirements(path: str) -> List[str]:
         # get rid of newlines
         line = line.replace('\n', '')
         # if version is specified (non-word chars present)
-        if re.search('\W', line):
+        if re.search('(~|=|<|>|;)', line):
```

Comment:
can you please avoid duplication of the same parser in different installation scripts? You can do it in separate PR.